docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -0,0 +1,373 @@
+---
+title: Telemetry (alpha)
+---
+
+## Kyma's telemetry component
+
+The focus of Kyma's observability functionality is the pre-integrated collection of telemetry data from the users' workloads, and making that data available for further analysis and storage in backends of any kind and location. The telemetry component provides the configurable collection and shipment components, and with that, separates them explicitly from the storage and analysis in a specific backend system. Lightweight in-cluster backends can be installed with dedicated observability components.
+
+For now, the telemetry component supports the logging domain only. You get a log collector (Fluent Bit) and you can configure the log shipment with external systems using runtime configuration with a dedicated Kubernetes API (CRD). With that, for example, you can integrate with vendors like [VMWare](https://medium.com/@shrishs/log-forwarding-from-fluent-bit-to-vrealizeloginsightcloud-9eeb14b40276) using generic outputs, or with any vendor via a [fluentd integration](https://medium.com/hepsiburadatech/fluent-logging-architecture-fluent-bit-fluentd-elasticsearch-ca4a898e28aa) using the forward output. Kyma's optional `logging` component complements the telemetry component, providing `Loki` as pre-configured log backend.
```

Comment:
Our [guidelines](https://kyma-project.io/community/guidelines/content-guidelines/02-style-and-terminology/#terminology) don't like phrasing it this way...

<img width="753" alt="image" src="https://user-images.githubusercontent.com/76950046/179961605-388230fe-b500-457e-86d9-12b36adbfab8.png">

Reply:
What do you suggest

Reply:
How about this:

> To support the logging domain, the telemetry component provides a log collector, Fluent Bit. You can configure the log shipment with external systems using runtime configuration with a dedicated Kubernetes API (CRD). With that, for example, you can integrate with vendors like...